distr_tracing/distr_tracing_rn/distr-tracing-rn-2-9-2.adoc

```diff
@@ -0,0 +1,137 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-2-9-2"]
+= Release notes for {DTProductName} 2.9.2
+:context: distributed-tracing-rn-2-9-2
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+[id="component-versions_distributed-tracing-rn-2-9-2"]
+== Component versions in the {DTProductName} 2.9.2
+
+[options="header"]
+|===
+|Operator |Component |Version
+|{JaegerName}
+|Jaeger
+|1.47.0
+
+|{OTELName}
+|OpenTelemetry
+|0.81.0
+
+|{TempoName}
+|Tempo
+|2.1.1
+|===
+
+== CVEs
+
+This release fixes link:https://bugzilla.redhat.com/show_bug.cgi?id=2245180[CVE-2023-45142].
```

Comment:
Bugzilla link for fixed CVE-2023-46234 https://bugzilla.redhat.com/show_bug.cgi?id=2246470

Reply:
Thank you, Ishwar. Already applied.